tools/accuracy_checker/openvino/tools/accuracy_checker/annotation_converters/kitti_converter.py

```diff
@@ -0,0 +1,94 @@
+            ),
+            'images_suffix': StringField(optional=True, default='.png', description='Suffix for images'),
+        })
+        return params
+
+    def configure(self):
+        self.images_dir = self.get_value_from_config('images_dir')
+        self.annotation_dir = self.get_value_from_config('annotation_dir')
+        self.labels_file = self.get_value_from_config('labels_file')
+        self.label_start = self.get_value_from_config('label_start')
+        self.images_suffix = self.get_value_from_config('images_suffix')
+
+    def convert(self, check_content=False, progress_callback=None, progress_interval=100, **kwargs):
+        content_errors = None if not check_content else []
+        ann_files = self.annotation_dir.glob('*.txt')
+        image_ann_pairs = [(ann.name.replace('.txt', self.images_suffix), ann.name) for ann in ann_files]
+
+        meta, self.reversed_label_map = self.get_meta()
+        num_iterations = len(image_ann_pairs)
+        annotations = []
+        for idx, (identifier, annotation_files) in enumerate(image_ann_pairs):
+            labels, x_mins, y_mins, x_maxs, y_maxs = self.parse_annotation(annotation_files)
+            annotations.append(DetectionAnnotation(identifier, labels, x_mins, y_mins, x_maxs, y_maxs))
+            if check_content:
+                if self.images_dir is None:
+                    self.images_dir = self.annotation_dir.parent / 'image_2'
+                check_file_existence(self.images_dir / identifier)
+                content_errors.append('{}: does not exist'.format(self.images_dir / identifier))
+            if progress_callback and idx % progress_interval == 0:
+                progress_callback(idx * 100 / num_iterations)
+
+        return ConverterReturn(annotations, meta, content_errors)
+
+    def parse_annotation(self, annotation_file):
+        labels, x_mins, y_mins, x_maxs, y_maxs = [], [], [], [], []
+        for line in read_txt(self.annotation_dir / annotation_file):
+            annotation_line = line.split()
+            label = annotation_line[0].lower()
+            if not self.reversed_label_map.get(label):
+                continue
+            x_min, y_min, x_max, y_max = annotation_line[4:8]
+            labels.append(self.reversed_label_map[label])
+            x_mins.append(float(x_min))
+            y_mins.append(float(y_min))
+            x_maxs.append(float(x_max))
+            y_maxs.append(float(y_max))
+        return np.array(labels), np.array(x_mins), np.array(y_mins), np.array(x_maxs), np.array(y_maxs)
+
+    def get_meta(self):
+        labels = read_txt(self.labels_file)
+        reversed_label_map = {}
+        label_map = {}
+        for idx, label_name in enumerate(labels, start=self.label_start):
+            label_map[idx] = label_name.lower()
+            reversed_label_map[label_name.lower()] = idx
+        meta = {'label_map': label_map}
+        return meta, reversed_label_map
```

Comment:
is it possible return only meta in this method?
get_meta used as part of api for getting dataset info without conversion, iy will be strange if somebody uses it and get not only meta but something else?

Reply:
Done